### DISCUSSION TITLE
Need some help

### DISCUSSION THREAD

**Discussion by Majboor** [jyjblrd/Mocap-Drones#13] [General]
I have been looking for such mini drone solution, is it possible that you could explain the process of linking the esp32 or esp wifi with the  F3 EVO Micro. 
I am working on a project to make several of these drones and inter link them in a channel similar to nrf modules I could link these through the network. I would really appreciate any effort in explaining the process
Thanks

**Comment by jyjblrd** [comment]
Hi, I used the sbus arduino library to send information from the ESP32 to the flight controller.

The sbus library can be found here: https://github.com/bolderflight/sbus/
You can find an example of its usage here: https://github.com/bolderflight/sbus/blob/main/examples/arduino/sbus_example/sbus_example.ino

Within my own project, all the sbus stuff is within the receiver esp32's code: https://github.com/jyjblrd/Mocap-Drones/blob/main/receiver_esp32/receiver_esp32.ino
I set the pwm data on [these lines](https://github.com/jyjblrd/Mocap-Drones/blob/7857ac0c0918948846a084a2c50841a76bc385dd/receiver_esp32/receiver_esp32.ino#L245C1-L248C23), and then send the data on [this line](https://github.com/jyjblrd/Mocap-Drones/blob/7857ac0c0918948846a084a2c50841a76bc385dd/receiver_esp32/receiver_esp32.ino#L257C1-L258C21).

Hope that helps!
